_includes/feedback.html

```diff
@@ -4,11 +4,12 @@
     Was this article helpful? <a class="link-burrito" href="#yes">Yes</a>  - <a class="link-burrito" href="#no">No</a>
   </p>
   <p class="feedback-no">
-    Well, that's awkward <i class="em em-sweat_smile"></i>. Would you mind
```

Comment:
Oops I actually liked this part. My bad - I should have ben more specific and included the intro bit.